src/Engine/System/SoundSystem.cpp

```diff
@@ -0,0 +1,80 @@
+#include "SoundSystem.hpp"
+
+const Vector<SoundComponent *> & SoundSystem::s_soundComponents(Scene::getComponents<SoundComponent>());
+String SoundSystem::SOUND_DIR = "../resources/soundeffects/";
```

Comment:
This should be in relation to EngineApp::RESOURCE_DIR

Reply:
Changed.